tests/integration/025_notifications/main.tf

```diff
@@ -18,9 +18,14 @@ resource "env0_notification" "test_notification_2" {
   value = "https://someurl2.com"
 }
 
-data "env0_notifications" "all_notifications" {}
+data "env0_notifications" "all_notifications" {
+  depends_on = [env0_notification.test_notification_1, env0_notification.test_notification_2]
+}
+
+data "env0_notification" "test_notification_1" {
```

Comment:
why not just use 
```
data "env0_notification" "my_notification" {
  name = locals.test_notification_1_name
}
```
here?

Reply:
we wish to see that the `all_notifications` works as expected too, so I suggest we utilize its data, so if it got broken for some reason the test will fail

Reply:
agree with @avnerenv0 .
can be replaced with post condition on the `all_notifications`

Reply:
[0c08f76](https://github.com/env0/terraform-provider-env0/pull/735/commits/0c08f76aa77d2f10d0a9144aa5caf8222804f19b)

used the resource name, and used output for the `all_notifications` verification, that it got one of the notifications